FirebaseVertexAI/Sources/GenerateContentResponse.swift

```diff
@@ -88,6 +88,26 @@ public struct GenerateContentResponse: Sendable {
     }
   }
 
+  /// Returns inline data parts found in any `Part`s of the first candidate of the response, if any.
+  public var inlineDataParts: [InlineDataPart] {
+    guard let candidate = candidates.first else {
+      VertexLog.error(
+        code: .generateContentResponseNoCandidates,
+        "Could not get inline data parts because the response has no candidates. The accessor only checks the first candidate."
+      )
```

Comment:
nit: > 100 chars wide. Consider using multi-line string.
```suggestion
      VertexLog.error(
        code: .generateContentResponseNoCandidates,
        "Could not get inline data parts because the response has no candidates. The accessor only checks the first candidate."
      )
```